geoutils/georaster/raster.py

```diff
@@ -195,31 +210,97 @@ def __init__(
 
         :return: A Raster object
         """
+        self.driver: str | None = None
```

Comment:
What is the benefit of defining the Class attributes + type hints inside `__init__` instead of outside here?

Reply:
Uuh probably no difference. There's a difference if no default argument is given:
```python
class Raster:
    driver: str | None   # This is only a type signature

Raster().driver   # Will raise an error because driver does not exist (but the expected type is known)
```
But when a default is defined, I honestly don't know the difference.